apstools/beamtime/apsbss.py

```diff
@@ -167,7 +167,7 @@ def epicsUpdate(prefix):
     bss.status_msg.put("clearing PVs ...")
     bss.clear()
 
-    cycle = bss.esaf.aps_cycle.get()
+    cycle = bss.esaf.aps_cycle.get(timeout=10)
```

Comment:
Not a good idea to add an explicit timeout here in library code.  Better to set an overall timeout from the caller.

Reply:
[For example](https://github.com/BCDA-APS/bluesky_training/blob/main/bluesky/instrument/framework/initialize.py#L125-L128), this must be called before any ophyd instances are created:

```python
# set default timeout for all EpicsSignal connections & communications
EpicsSignalBase.set_defaults(
    auto_monitor=True, timeout=60, write_timeout=60, connection_timeout=60,
)
```